python/seldon_core/microservice.py

```diff
@@ -60,7 +63,7 @@ def start_servers(
     """
     p2 = None
     if target2:
-        p2 = mp.Process(target=target2, daemon=True)
```

Comment:
Why this change?

Reply:
https://github.com/SeldonIO/seldon-core/pull/3356#discussion_r661828299